src/extension/options-page/DashboardDialogs/Profile.tsx

```diff
@@ -0,0 +1,241 @@
+import { ProfileIdentifier } from '../../../database/type'
+import Services from '../../service'
+import { getNetworkWorker } from '../../../social-network/worker'
+import { geti18nString } from '../../../utils/i18n'
+
+const useStyles = makeStyles(theme =>
+    createStyles({
+        button: {
+            marginTop: theme.spacing(1),
+            marginBottom: theme.spacing(1),
+            width: 140,
+        },
+        buttonLarge: {
+            width: 240,
+        },
+    }),
+)
+
+interface ProfileConnectTestSuccessDialogProps {
+    identifier: ProfileIdentifier
+    nickname: string
+    onClose: () => void
+}
+
+function ProfileConnectTestSuccessDialog(props: ProfileConnectTestSuccessDialogProps) {
+    const { identifier, nickname, onClose } = props
+    return (
+        <Dialog open onClose={onClose}>
+            <DialogContentItem
+                simplified
+                title={geti18nString('setup_successful')}
+                content={geti18nString('dashboard_bio_test_succeeful', [
+                    identifier.userId,
+                    identifier.network,
+                    nickname,
+                ])}
+                actions={
+                    <ActionButton variant="outlined" color="default" onClick={onClose}>
+                        {geti18nString('ok')}
+                    </ActionButton>
+                }></DialogContentItem>
+        </Dialog>
+    )
+}
+
+interface ProfileConnectTestFailedDialog {
+    provePost: string
+    userId: string
+    onClose: () => void
+}
+
+function ProfileConnectTestFailedDialog(props: ProfileConnectTestFailedDialog) {
+    const { provePost, userId, onClose } = props
+    return (
+        <Dialog open onClose={onClose}>
+            <DialogContentItem
+                simplified
+                title={geti18nString('setup_failure')}
+                content={geti18nString('dashboard_bio_test_failed', [provePost, userId])}
+                actions={
+                    <ActionButton variant="outlined" color="default" onClick={onClose}>
+                        {geti18nString('ok')}
+                    </ActionButton>
+                }></DialogContentItem>
+        </Dialog>
+    )
+}
+
+interface ProfileConnectStartDialogProps {
+    nickname?: string
+    network?: string
+    onConfirm(name: string): void
+    onDecline(): void
+}
+
+export function ProfileConnectStartDialog(props: ProfileConnectStartDialogProps) {
+    const { nickname, network, onConfirm, onDecline } = props
+    const [name, setName] = useState('')
+
+    const content = (
+        <div style={{ alignSelf: 'stretch', textAlign: 'center', width: '100%' }}>
+            <TextField
+                style={{ width: '100%', maxWidth: '320px' }}
+                autoFocus
+                variant="outlined"
+                value={name}
+                onChange={e => setName(e.target.value)}
+                label={`Username on ${network}`}></TextField>
```

Comment:
Move it to `message.json`?

Reply:
if it should have its translation (or should not)

Reply:
if it is user visible, or an `aria`

Reply:
It should have a translation